src/index.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react'
-import kebabCase from 'lodash.kebabcase'
-import transform from 'lodash.transform'
-import isString from 'lodash.isstring'
-import isNumber from 'lodash.isnumber'
+import get from 'lodash.get'
+import merge from 'lodash.merge'
+import isString from 'lodash.isString'
+import memoize from 'lodash.memoize'
+import isPlainObject from 'lodash.isPlainObject'
```

Comment:
Hm, while this may work, I prefer the actual package name in the `from` part, because it represents a file system path. While using the capitalized paths will work on OSX and many other systems, I think it is less confusing and more secure to use the actual path names.

Reply:
Absolutely correct. Didn't even think about it, thanks!